### DISCUSSION TITLE
Update check fails.

### DISCUSSION THREAD

**Discussion by Patrick15745** [pretix/pretix#3205] [Q&A]
The last update check was not successful. The pretix.eu server was not accessible.
https://pretix.domain.de/control/global/update/ 

We are talking about a self-hosted version. The cronjob is set up. Internal firewall is set. Update directly in Linux works. We are behind a university proxy that could be the problem. The logs tell me it is not a pretix problem. 

Which ports are necessary? Could Apache /Nginx settings "reverse proxy" block something?

Browser said: 
_Proxy Error
The proxy server received an invalid response from an upstream server.
The proxy server could not handle the request
Reason: Error reading from remote server
Apache/2.4.52 (Ubuntu) Server at pretix.th-wildau.de Port 443_

**Comment by raphaelm** [comment]
The server needs to be able to reach https://pretix.eu at port 443 for the update check to work
